webapp/www/templates/home.html

```diff
@@ -45,43 +45,43 @@
 <fullpage class="instant-size" page-class="page">
```

Comment:
As I said in the high level comment, since this is cards anyway, maybe this can be a list instead of fullpage?
That should help with the scrolling on firefox and also resolve #281.
Or maybe the solution is to use a different fullpage library?